integrationtests/src/test/java/org/axonframework/integrationtests/eventhandling/TrackingEventProcessorTest.java

```diff
@@ -806,6 +818,8 @@ void testResetBeforeStartingPerformsANormalRun() throws Exception {
         assertWithin(1, TimeUnit.SECONDS, () -> assertEquals(4, handled.size()));
         assertEquals(0, handledInRedelivery.size());
         assertFalse(testSubject.processingStatus().get(0).isReplaying());
+        assertTrue(testSubject.processingStatus().get(0).getReplayPosition().isPresent());
```

Comment:
Shouldn't both `getReplayPosition` and `getResetTokenPosition` return false it this point, as `isReplaying` is false too?